### PR TITLE
fixed travis on osx and verbose test output

### PR DESCRIPTION
**Pull request by markwellis**
there is no perl 5.22 on travis, that's why i removed it

### CODE REVIEW

**Comment by markwellis** [comment]
also i don't upgrade brew, so the osx builds are faster

**Comment by markwellis** [comment]
since it's only travis changes i'm merging this.

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 sudo: false
 language: perl
 perl:
-  - "5.22"
```
Comment (manchicken) [inline .travis.yml]: We need to add 5.22 back in, no?

File: .travis.yml [hunk 2]
```diff
@@ -28,9 +26,8 @@ before_script:
...
-  - export MQHOST="$LOCAL_MQHOST"
-  - export MQUSERNAME="guest"
-  - export MQPASSWORD="guest"
-  - if [[ "$MQHOST" != "" ]]; then cover -test -report coveralls -ignore_re "\.c|\.h"; fi
-  - if [[ "$MQHOST" != "" ]]; then perl -I blib/lib -I blib/arch xt/100_transaction_memory_leak.t; fi
-  - if [[ "$MQHOST" != "" ]]; then perl -I blib/lib -I blib/arch xt/101_headers_memory_leak.t; fi
+  - if [[ "$(which brew)" == "" ]]; then export MQHOST="localhost"; export MQUSERNAME="guest"; export MQPASSWORD="guest"; fi
+  - cover -make "make TEST_VERBOSE=1" -test test -report coveralls -ignore_re "\.c|\.h"
+  - export MQHOST="localhost"; export MQUSERNAME="guest"; export MQPASSWORD="guest"
+  - if [[ "$(which brew)" == "" ]]; then perl -I blib/lib -I blib/arch xt/100_transaction_memory_leak.t; fi
```
Comment (manchicken) [inline .travis.yml]: I was trying not to tie the exception only to OSX... but I guess Travis only gives us OSX and GNU/Linux.

File: .travis.yml (hunk 1 above)
Comment (markwellis) [inline .travis.yml]: Travis doesn't have 5.22, so it just runs 5.14 iirc, which we already test

File: .travis.yml [hunk 3]
```diff
@@ -28,9 +26,8 @@ before_script:
...
   - make
 script:
-  - export MQHOST="$LOCAL_MQHOST"
-  - export MQUSERNAME="guest"
-  - export MQPASSWORD="guest"
-  - if [[ "$MQHOST" != "" ]]; then cover -test -report coveralls -ignore_re "\.c|\.h"; fi
-  - if [[ "$MQHOST" != "" ]]; then perl -I blib/lib -I blib/arch xt/100_transaction_memory_leak.t; fi
-  - if [[ "$MQHOST" != "" ]]; then perl -I blib/lib -I blib/arch xt/101_headers_memory_leak.t; fi
+  - if [[ "$(which brew)" == "" ]]; then export MQHOST="localhost"; export MQUSERNAME="guest"; export MQPASSWORD="guest"; fi
+  - cover -make "make TEST_VERBOSE=1" -test test -report coveralls -ignore_re "\.c|\.h"
```
Comment (manchicken) [inline .travis.yml]: Weeeee, great stuff.

**Pull request merged**